packages/headless/src/modules/order/services/customer.service.ts

```diff
@@ -32,6 +37,40 @@ export class OrderCustomerService {
     );
   }
 
+  async reOrder(userId: string, body: ReOrderQuery): Promise<any>{
+    let { ignoreInvalidItems, overWriteCart, orderId } = body; 
+
+    const order = await this.orderRepository.getOrderById(orderId);
```

Comment:
also check if the order was created by this user or not. i mean find the order by order id and user id